packages/profile-metrics-controller/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.1]
```

Comment:
Nit: Should this be minor, if it's not a bug fix?

Reply:
Done in 0ade03b38a1a